.github/workflows/size-impact/report-size-impact.js

```diff
@@ -1,9 +1,32 @@
-import { reportSizeImpactIntoGithubPullRequest } from "@jsenv/github-pull-request-filesize-impact";
+import {
+  reportFileSizeImpact,
+  readGithubWorkflowEnv,
+  none,
+  gzip,
+  brotli,
+} from "@jsenv/file-size-impact";
 
-reportSizeImpactIntoGithubPullRequest({
-  projectDirectoryUrl: new URL("../../../", import.meta.url),
-  baseSnapshotFileRelativeUrl: process.argv[2],
-  headSnapshotFileRelativeUrl: process.argv[3],
-  commentSections: { fileByFileImpact: true },
-  generatedByLink: true,
+reportFileSizeImpact({
+  ...readGithubWorkflowEnv(),
+
+  trackingConfig: {
+    browser: {
+      "./dist/*": true,
+      "./dist/*.map": false,
+      "./dist/system-node.cjs": false,
+    },
+    node: {
+      "./dist/system-node.cjs": true,
+    },
+    extras: {
+      "./dist/extras/**/*.js": true,
+      "./dist/extras/**/*.map": false,
+    },
+  },
+  transformations: { none, gzip, brotli },
+  commentSections: {
+    overallSizeImpact: false,
```

Comment:
overallSizeImpact is still disabled because it would consider minified files too. We could enable it but we would have to create a dedicated group in `trackingConfig` for minified files if we do so.